README.md

```diff
@@ -38,7 +38,14 @@ Cases where the RFC process need *not* be used:
    to 2 weeks.
  - If substantive disagreement remains, then the issue is escalated to the
    [core team](https://github.com/orgs/web-platform-tests/teams/wpt-core-team/)
-   for a decison. Only arguments that have already been made on the issue may
-   be taken into account. As a last resort, the core team may decide using a
-   &geq;2/3 majority vote. A decision should be made within 1 additional week.
- - Close the RFC issue when it has been implemented.
+   for a decison:
+   - Only arguments that have already been made on the issue may be taken into
+     account.
+   - If meaningful new information is presented, the comment period can be
+     extended to ensure that all participants have time to consider it.
+   - When no new information is forthcoming, a decision should be made within 1
+     additional week.
+   - The decision can be to defer the issue until a later time. In this case,
+     a new RFC should be sent when revisiting the issue.
```

Comment:
Why is a new RFC required?

Reply:
Otherwise we'll leave RFCs open even though it's unclear if they will actually be revisited, and when they are people who subscribe to the repo won't be notified of a new discussion and are less likely to notice.

But those are GitHub details and I can just delete this sentence.